ngraph/frontend/paddlepaddle/src/op/hard_swish.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "hard_swish.hpp"
+#include <ngraph/opsets/opset6.hpp>
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs hard_swish(const NodeContext& node)
+                {
+                    auto data = node.get_ng_input("X");
+                    if (node.has_attribute<float>("threshold"))
+                    {
+                        auto threshold = node.get_attribute<float>("threshold");
+                        PDPD_ASSERT(std::abs(threshold - 6.0) < 0.001,
```

Comment:
I have some doubts about this threshold. Consider model like
```
x1  = hard_swish(threshold=6.0, scale=3.0, offset=3.0)
x2  = hard_swish(threshold=6.0001, scale=3.0001, offset=3.0001)
out = (x2 - x1) * 10000
```
Execution of this will produce significant non-null values. But in OV it will be always 0

Reply:
The OV hard_swish only supports fixed value of threshold(6.0) and offset(3.0). The check here intends to ensure that the Paddle Ops have those default value. Moreover, we don't plan to support other value because the Paddle has also fixed their threshold to 6 and offset to 3 in [API 2.1](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/hardswish_en.html).